src/main/java/com/easyrest/components/AddRestaurant.java

```diff
@@ -0,0 +1,43 @@
+package com.easyrest.components;
+
+import com.easyrest.pages.CustomerPage;
+import com.easyrest.pages.customerPanel.MyRestaurants;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class AddRestaurant {
+
+    private WebDriver driver;
+    private final By restaurantNameField = By.xpath("//input[@name='name']");
+    private final By restaurantAddressField = By.xpath("//input[@name='address']");
+    private final By addButton = By.xpath("//*[text() = 'Add']");
+    private final By cancel = By.xpath("//*[text() = 'Cancel']");
+
+    public AddRestaurant(WebDriver driver){
+        this.driver=driver;
+    }
+
+
+    public AddRestaurant enterRestaurantName(String name) {
+        driver.findElement(restaurantNameField).clear();
```

Comment:
code that find element and clear field is repeated several times. I think we should create method that will clear the field in BasePage.